common/src/main/java/org/dash/wallet/common/services/analytics/AnalyticsConstants.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2021 Dash Core Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dash.wallet.common.services.analytics
+
+object AnalyticsConstants {
+    const val CALLING_ACTIVITY = "calling_activity"
```

Comment:
looks like this file will have a merge conflict with #737.

Reply:
that's ok, merge one - I'll fix the other.